### ISSUE TITLE
multi-module project support

### ISSUE THREAD

**Issue by afollestad** [theapache64/gpm#23]
Something like this:

```gradle
gpm app:install okhttp
gpm common:utilities:install okhttp
```

It could probably simple search for a build.gradle in the given path

**Comment by theapache64** [comment]
@afollestad  Alright. that'd be very useful. But the command would be something like this

```
gpm install :app okhttp
gpm install :common:utilities okhttp
```
where arguments starting with `:` would be considered as directory/module.

The reason  `install` is a command, and `Picocli` can only accept arguments after the command name. 

Would this work?

**Comment by theapache64** [comment]
@afollestad 

Done! 

The syntax would be 

```
gpm :module install <depName>
gpm :moduleDir:module <depName>
```

eg:

```
gpm :features:auth install okhttp
```

This feature will be available from `1.0.4-alpha01` :)
